.gitignore

```diff
@@ -0,0 +1,5 @@
+﻿################################################################################
```

Comment:
Do we need this gitignore?

Reply:
For testing sake, I loaded only one and then a few a few meshes in the folder instead of the whole animation (It takes about 10-12 minutes to load 4 meshes while I debug), I won't include that .gitignore in merge.